ras_party/controllers/account_controller.py

```diff
@@ -477,26 +478,26 @@ def add_new_survey_for_respondent(payload, tran, session):
     # This ensures the log message is only written once the DB transaction is committed
     tran.on_success(lambda: logger.info('Respondent has enroled to survey for business',
                                         survey_name=survey_name,
-                                        business=business_id))
+                                        business=str(business_id)))
```

Comment:
`business_id` comes from the response from the case service, so should already be a string